### DISCUSSION TITLE
Make Wiki page accessable to everyone

### DISCUSSION THREAD

**Discussion by hansotto1** [requarks/wiki#7084] [Help / Questions]
Hello,

we configured our Wiki to a closed user group that need to log in to write and read pages. But anyway I would gnerate a free accessable Wiki page that can be opened without the need to log in. Can this be realized with the page- and/or group rights?

**Comment by yoitguy** [comment]
Absolutely. I use it the same way. We keep departmental documentation locked down but we have an all-users facing section. Grant the "Guests" group permissions to read:pages, read:assets, and read:comments access. Also give "Guests" Page Rules access to read:pages, read:assests, read:comments for the path that you want them to have access to. Also under "Guests" Page Rules, create deny rules for all other paths.
